config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   get "/", to: 'welcome#index', as: :welcome
   # resources :welcome, :path => '/', only: :index, as: :welcome
-
+  get "/register", to: "users#new"
+  post "/users", to: "users#create"
+  get "/dashboard", to: "users#show"
```

Comment:
I smell a name spacing coming on!  What do you think we should do when integrating authentication and authorization of our users and their API key.  I saw some documentation about how to load things to a session, I'm going to be really excited to figure out that next step.

Reply:
Per our conversation: We don't have to deal with authorization, but we might need to consider simply our authentication process and no one who is authenticated can see anything.

Reply:
Sessions are in the Welcome Branch! I'm glad to see you predicting that! Confirms that my methodology is matching yours :)

Reply:
We absolutely need to write in tests for logged out users getting 400 errors :)